startup-scripts/udm-utilities/on_boot.d/40-udp-proxy-2020.sh

```diff
@@ -24,13 +26,20 @@ if [ -z "$CACHETTL" ]; then
     CACHETTL=90
 fi
 
+if [ "$ARG1" = "clean" ]; then
+    if podman container exists ${CONTAINER}; then
+        podman stop ${CONTAINER}
+        podman container rm ${CONTAINER}
+    fi
```

Comment:
this seems a little confusing... the `clean` command does the clean, but then also starts because the script continues executing at this point?